src/main/frontend/extensions/code.cljs

```diff
@@ -294,13 +294,15 @@
      (when-not (= mode "calc")
        [:div.extensions__code-lang
         (string/lower-case mode)]))
-   [:textarea (merge {:id id
-                      ;; Expose the textarea associated with the CodeMirror instance via
-                      ;; ref so that we can autofocus into the CodeMirror instance later.
-                      :ref textarea-ref-name
-                      :default-value code} attr)]
-   (when (= (:data-lang attr) "calc")
-     (calc/results (:calc-atom state)))])
+   [:div.flex.flex-1.flex-row
```

Comment:
This changes the code editor's style.
Under light mode:

<img width="531" alt="image" src="https://user-images.githubusercontent.com/72891/153210380-74304c95-15aa-4443-ab10-b236d15aeadc.png">